torii/_typing.py

```diff
@@ -1,8 +1,8 @@
 # SPDX-License-Identifier: BSD-2-Clause
+from __future__ import annotations
```

Comment:
Per the other files here, this should space spaced one line off the SPDX license ident

Reply:
The `from __future__ import annotations` are a special exception generally due to their magic, they always must be the first thing in the file (excluding doc comments)

I guess we /could/ space them down a line, *should* be fine:tm:

Reply:
It would definitely be more consistent to, despite being a special import